storage_service/locations/api/resources.py

```diff
@@ -592,6 +595,30 @@ def recover_aip_request(self, request, bundle, **kwargs):
         return http.HttpResponse(status=status_code, content=response_json,
             content_type='application/json')
 
+    @_custom_endpoint(expected_methods=['post'],
+        required_fields=('location_uuid', ))
+    def move_request(self, request, bundle, **kwargs):
+        """
+        Move package to another location
+        """
+        request_info = bundle.data
+        package = bundle.obj
+
+        if package.status == Package.MOVING:
+            message = "Move already in progress."
+            success = False
+        else:
```

Comment:
@Hwesta @jhsimpson would it be too restrictive to allow only moves of packages in the UPLOADED state? Asking because the code seems to be setting the state to UPLOADED if the move was successful, so probably this state was assumed? ( https://github.com/artefactual/archivematica-storage-service/pull/143/files#diff-5f2ead45b8622f27e9b0bf6333334fffR380 )